docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,601 @@
+
+  ```
+  cd tls-ca && mkdir tls admin
+  cd ../org1-ca && mkdir tls admin
+  cd ../int-ca && mkdir tls admin
+  ```
+
+  The resulting folder structure resembles:
+    ```
+    fabric-ca-client
+      ├── int-ca
+      │   ├── admin
+      │   └── tls
+      ├── org1-ca
+      │   ├── admin
+      │   └── tls
+      └── tls-ca
+          ├── admin
+          └── tls
+    ```      
+
+**Tip:** The first time you issue an `enroll` command from the CA client, if the `fabric-ca-client-config.yaml` does not already exist in the `$FABRIC_CA_CLIENT_HOME` directory, it is generated. When you customize the values in this file, they are used automatically by the CA client and do not have to be passed on the command line on a subsequent `enroll` command.
+
+You can also use environment variables to specify certificate locations.
+- `FABRIC_CA_CLIENT_HOME` - Specify the path to where the generated certificates should be stored and which admin certificates should be used when users are registered and enrolled with a CA. This path is relative to the Fabric CA client binary.
+- `FABRIC_CA_CLIENT_TLS_CERTFILES` - Specify the location and name of the TLS certificate. If the path of the environment variable `FABRIC_CA_CLIENT_TLS_CERTFILES` is not an absolute path, it will be parsed as relative to the Fabric CA client’s home directory as specified by `FABRIC_CA_CLIENT_HOME`.
+- `FABRIC_CA_CLIENT_MSPDIR` - While you can use this environment variable to specify the name of the folder under `FABRIC_CA_CLIENT_HOME` where the generated certificates are stored, _a better option when you do not want the certificates to be stored in the default `msp` folder is to pass the `-M` flag on the enroll command itself and explicitly specify the location for the generated certificates._ If not specified on the command, the location defaults to `$FABRIC_CA_CLIENT_HOME/msp`. **Important:** When multiple members are enrolled with the CA, **you need to store their certificates in distinct folders** in order to differentiate them.  
+
+Throughout these instructions, we modify the settings in the CA server configuration .yaml file and then override the settings with flags on the Fabric CA client CLI commands as needed. For example the register and enroll commands use the `--tls.certfiles` flag to specify the location of the TLS certificate. Because we have a single instance of the Fabric CA client binary that is used to interact with the three different CAs, (TLS CA, organization CA, intermediate CA), we use the `FABRIC_CA_CLIENT_HOME` environment variable to designate the location of the certificate folders for the targeted CA.
```

Comment:
"we use the `FABRIC_CA_CLIENT_HOME` environment variable to designate the location of the certificate folders"

Isn't there a SINGLE home directory? if so, how could the single home directory designate the location of each of the three targeted CAs?
And which 'certificate folders' are you specifically talking about? The folders for the newly generated certificates? I thought the above paragraph said -M flag should be used to designate these folders.
Or maybe you mean FABRIC_CA_CLIENT_HOME should be updated on every call depending on which of the three CAs you are interacting with?

Reply:
I refactored the instructions to use the location of the Fabric CA Client binaries as the FABRIC_CA_CLIENT_HOME. (Note this is different that how it is used in the Operations Guide).
Then I use the --mspdir flag on the register and enroll commands to control which crypto material to use on the register command and which folder to store the generated certificates on the enroll command. 
It is a much cleaner pattern and avoids have to switch the value of the FABRIC_CA_CLIENT_HOME variable when you switch which CA the client is interacting with.

Reply:
looks good.